vertica/assets/monitors/vertica_replication_safety.json

```diff
@@ -0,0 +1,34 @@
+{
+	"name": "[Vertica] Nodes down above K-safety level",
+	"type": "query alert",
+	"query": "max(last_5m):avg:vertica.node.down{*} - avg:vertica.ksafety.current{*} > 1",
+	"message": "Number of nodes down is above the K-safety, which may lead to data becoming unavailable.",
+	"tags": [
+		"integration:vertica"
+	],
+	"options": {
+		"notify_audit": false,
+		"locked": null,
+		"timeout_h": 0,
+		"new_host_delay": 300,
+		"require_full_window": false,
+		"notify_no_data": false,
+		"renotify_interval": "0",
+		"renotify_occurrences": null,
+		"renotify_statuses": null,
+		"escalation_message": "",
+		"no_data_timeframe": null,
+		"include_tags": true,
+		"thresholds": {
+			"critical": 1,
+			"warning": 0,
+			"critical_recovery": 0,
+			"warning_recovery": 0
+		}
+	},
+	"priority": null,
+	"restricted_roles": null,
+	"recommended_monitor_metadata": {
+		"description": "Get notified when the query load is high."
```

Comment:
I think you should update this description too

Reply:
whoops 😳

Reply:
don't worry, I won't tell anyone where I saw this exact same description 😁